### PR TITLE
Updated workflows

### PR DESCRIPTION
**Pull request by scottx611x**
Includes our updated Workflows from Release 1.6.0

- [x] Add Updated WF names
- [x]  Update `repository_dependencies.xml`
- [x] Add updated WF descriptions
- [x] Add updated input file names/descriptions

### CODE REVIEW

File: chipseq_workflows/repository_dependencies.xml [hunk 1]
```diff
@@ -4,4 +4,5 @@
   <repository toolshed="toolshed.g2.bx.psu.edu" name="bowtie_wrappers" owner="devteam" changeset_revision="df86f29bedee" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="sam_to_bam" owner="devteam" changeset_revision="d04d9f1c6791" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="macs2" owner="iuc" changeset_revision="6d4babad010f" />
+  <repository toolshed="testtoolshed.g2.bx.psu.edu" name="split_paired_collection" owner="scottx611x" changeset_revision="429cae20b148" />
```
Comment (hackdna) [inline chipseq_workflows/repository_dependencies.xml]: This tool should be published in https://toolshed.g2.bx.psu.edu/view/refinery-platform and the source code should be in https://github.com/refinery-platform/galaxy-tools.

File: rnaseq_workflows/repository_dependencies.xml [hunk 2]
```diff
@@ -4,4 +4,5 @@
   <repository toolshed="toolshed.g2.bx.psu.edu" name="trimmomatic" owner="pjbriggs" changeset_revision="f8a9a5eaca8a" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="tophat2_with_gene_annotations" owner="scottx611x" changeset_revision="015a0cc3c012" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="featurecounts" owner="yhoogstrate" changeset_revision="e04fbcc4e91a" />
+  <repository toolshed="testtoolshed.g2.bx.psu.edu" name="split_paired_collection" owner="scottx611x" changeset_revision="429cae20b148" />
```
Comment (hackdna) [inline rnaseq_workflows/repository_dependencies.xml]: This tool should be published in https://toolshed.g2.bx.psu.edu/view/refinery-platform and the source code should be in https://github.com/refinery-platform/galaxy-tools.

**Review by hackdna — CHANGES_REQUESTED** [review]

**Comment by scottx611x** [comment]
@hackdna Yeah thanks, this isn't ready yet though. I'll request a review from you when it is. Sorry if I did request you earlier.

**Comment by hackdna** [comment]
OK. I replied because I've received a notice asking for a review of this PR.

**Comment by scottx611x** [comment]
@hackdna This is ok to review now

File: chipseq_workflows/repository_dependencies.xml [hunk 3]
```diff
@@ -4,4 +4,5 @@
   <repository toolshed="toolshed.g2.bx.psu.edu" name="bowtie_wrappers" owner="devteam" changeset_revision="df86f29bedee" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="sam_to_bam" owner="devteam" changeset_revision="d04d9f1c6791" />
   <repository toolshed="toolshed.g2.bx.psu.edu" name="macs2" owner="iuc" changeset_revision="6d4babad010f" />
+  <repository toolshed="toolshed.g2.bx.psu.edu" name="split_paired_collection" owner="scottx611x" changeset_revision="7432970f4f80" />
```
Comment (hackdna) [inline chipseq_workflows/repository_dependencies.xml]: As I mentioned earlier, this tool should be published in https://toolshed.g2.bx.psu.edu/view/refinery-platform

**Review by hackdna — CHANGES_REQUESTED** [review]

File: chipseq_workflows/repository_dependencies.xml (hunk 3 above)
Comment (scottx611x) [reply chipseq_workflows/repository_dependencies.xml]: I've granted the `refinery-platform` user write access to that repository. I don't have the ability to publish tools as `refinery-platform`

Comment (hackdna) [reply chipseq_workflows/repository_dependencies.xml]: I've provided you with access info.

Comment (scottx611x) [reply chipseq_workflows/repository_dependencies.xml]: @hackdna Okay so if the `refinery-platform` user is to be the primary owner of the `split-paired-collection` tool I would need to deprecate the current tool, and upload that same code under a different name than `split-paired-collection`. I think that Galaxy prefers what I've already done (making the `refinery-platform` user an admin of the repo) from what I read here: https://galaxyproject.org/toolshed/repository/#manage-repository-administrators

Comment (hackdna) [reply chipseq_workflows/repository_dependencies.xml]: Right, all Refinery-related tools should be in https://toolshed.g2.bx.psu.edu/view/refinery-platform then all individual accounts should be provided access to those repos as necessary.

**Comment by scottx611x** [comment]
@hackdna This is ok to review again
[Most recent changes](https://github.com/stemcellcommons/scc-galaxy-tools/pull/1/files/e316b723dd337f2118fd6d32501a4fa19d001c0e..d3def26e80726b0b93272b00141cd5a481b9baeb)

**Pull request merged**